src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  width: 200px;
+  height: 408px;
+}
+
+.product {
+  padding: 32px 16px 16px;
+
+  &__photo {
+    position: center;
+    width: 100%;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-accent;
+  margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: auto;
+
+    &-stars {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        background-image: url(../images/star.svg);
+        background-repeat: no-repeat;
+        background-position: center;
+      }
+    
+      & :nth-child(-n+4) {
```

Comment:
isn't fixed. you should work with all possible modifiers (0-5 stars)
https://github.com/mate-academy/layout_product-cards/pull/3963#pullrequestreview-1812684729